Python/ast_unparse.c

```diff
@@ -58,6 +59,24 @@ append_charp(_PyUnicodeWriter *writer, const char *charp)
         } \
     } while (0)
 
+static PyObject *
+replace_inf_value(PyObject *repr)
+{
+    PyObject *inf_str = PyUnicode_InternFromString("inf");
```

Comment:
I don't see the value of using intern here. PyUnicode_FromString() is enough:

```suggestion
    PyObject *inf_str = PyUnicode_FromString("inf");
```